vcr/api/vcr/v2/api.go

```diff
@@ -366,7 +371,17 @@ func (w *Wrapper) VerifyVP(ctx context.Context, request VerifyVPRequestObject) (
 		validAt = &parsedTime
 	}
 
-	verifiedCredentials, err := w.VCR.Verifier().VerifyVP(request.Body.VerifiablePresentation, verifyCredentials, false, validAt)
+	signerDID, err := credential.PresentationSigner(request.Body.VerifiablePresentation)
+	if err != nil {
+		return nil, fmt.Errorf("cannot determine subject of VP: %w", err)
+	}
+
+	allowUntrustedIssuers := true
```

Comment:
add a comment here that it should be removed in the (distant) future

Reply:
Once the param is removed from Verifier().VerifyVP() this should pop up. Perhaps we should start a tracking issue for all features that can be removed when did:nuts is phased out?